packages/core-database-sequelize/lib/connection.js

```diff
@@ -341,7 +341,7 @@ module.exports = class SequelizeConnection extends ConnectionInterface {
     // commented out as more use cases to be taken care of
     // this.walletManager.purgeEmptyNonDelegates()
 
-    this.walletManager.cleanAllWallets()
+    this.walletManager.cleanse()
```

Comment:
Maybe `clear` instead of cleanse? What do we use as a method name in the storage class?